Modules/StudyProgramme/classes/class.ilObjStudyProgramme.php

```diff
@@ -54,6 +54,7 @@ public function __construct($a_id = 0, $a_call_by_reference = true) {
 		global $DIC;
 		$tree = $DIC['tree'];
 		$ilUser = $DIC['ilUser'];
+		$this->g_webdir = $DIC->filesystem()->web();
```

Comment:
Can you explain what the `g_` stands for? It is a bit unclear to me.

Should this defined as member variable?

Reply:
This is an internal convention for `globals` we use at CaT. Should not have leaked to the ILIAS-core code.

You are also correct about adding it as member.

Thx!